test/fileFactories/DummyFileFactory.test.js

```diff
@@ -0,0 +1,17 @@
+import DummyFileFactory from '../../src/fileFactories/DummyFileFactory'
+import BaseFileFactory from '../../src/fileFactories/BaseFileFactory'
+
+describe("DummyFileFactory", () => {
+
+    test('the dummy file factory extends base file factory', () => {
+        expect(
+            new DummyFileFactory()
```

Comment:
We can use jest [`beforeAll`](https://jestjs.io/docs/en/setup-teardown.html#one-time-setup) to initialize the unit under test (DummyFileFactory) and just use that variable on the test itself

Reply:
Sounds cool even though I dont understand 100% 🤣 Lets iterate from here